scripts/common/host-packages.sh

```diff
@@ -95,6 +95,22 @@ function dpkg_install_host_packages() {
     _dpkg_install_host_packages "$dir" "$dir_prefix" "${packages[@]}"
 }
 
+function _dpkg_apt_get_status_and_maybe_fix_broken_pkgs() {
+    logStep "Checking package manager status"
+    if apt-get check status ; then
+        logSuccess "Status checked successfully. No broken packages were found."
+        return
+    fi
+
+    logWarn "Attempting to correct broken packages by running sudo apt-get install --fix-broken --yes"
+    apt-get install --fix-broken --yes
```

Comment:
Will this work with airgap installs?

Reply:
`apt-get install --fix-broken --yes` ask for the package manager try to fix by found, installing and etc the required dependencies. It will try to get from the local machine and from what is currently available in the index ( provided by the repos configured to be used by it)

It works exactly the same way for online or offline.
The only problem that we might to be faced here within that I can think about in the airgap bundle is:

- a) The apt-get index was updated when the machine was connected to the internet
- b) Then, you disconnect from the internet 
- c) The apt-get has a lot of versions and packages configured in the index that would only be retrieved by the internet

So, when we run this command that could be fail.

However, note that: 
- a) That would mean the offiline env is not configured correctly. That is not how works with offline envs. Offiline and online envs works very similar. Usually the only diff is that for an offiline env the package manager (host.list) will be configured to get the packages from a centralized hosting instead of the internet.
- b) We ONLY try to fix when we broke the package manager. That means, kURL install can only fail because it could not properly ensure that all host packages in the machine are properly installed as should be instead of finishes with success and does not let the users know about

I hope that answered all your questions and concerns. If not please, feel free to reach out. 

c/c @emosbaugh